src/foam/core/FObject.java

```diff
@@ -309,6 +309,7 @@ default FObject copyFrom(FObject obj) {
         }
       } catch (ClassCastException ignore) {
         System.err.println("FObject.copyFrom "+p.getName()+" "+ignore.getMessage());
+        ignore.printStackTrace();
```

Comment:
If we are going to do then use the following instead:
foam.nanos.logger.StdoutLogger.instance().warning(message, ignore);
Then it's output correctly regardless of what system we are running under.